src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+main {
+  height: fit-content;
+  width: fit-content;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
```

Comment:
Try to remove this width and rerun tests.

Reply:
Thank you, all tests are passed now.